src/toolbar_editor.hpp

```diff
@@ -31,14 +31,17 @@ class Toolbar_Editor : public QWidget, private Ui::Toolbar_Editor
     Q_OBJECT
     Q_PROPERTY(QMainWindow* targetWindow READ targetWindow WRITE setTargetWindow DESIGNABLE false)
     Q_PROPERTY(Qt::ToolButtonStyle buttonStyle READ buttonStyle WRITE setButtonStyle )
+    Q_PROPERTY(bool _customToolbarRemovalOnly READ customToolbarRemovalOnly WRITE setCustomToolbarRemovalOnly )
```

Comment:
The public name of the property shouldn't have the leading underscore, also this name is a bit long imo.